### ISSUE TITLE
Raycast can paste text to input

### ISSUE THREAD

**Issue by letungcntt** [flutter/flutter#153907]
### Use case

With MacOS, application Raycast, Alfred can't paste text to input. (The another application work)

https://github.com/user-attachments/assets/92706ceb-2499-4991-8f51-44a81829a184

### Proposal

Flutter can paste text by action other

**Comment by huycozy** [comment]
Additional information: I see this is working on Flutter Web app. It only doesn't work on Flutter macOS app. And [Raycast](https://www.raycast.com/) app seems to be available on macOS only.

**Comment by letungcntt** [comment]
> Additional information: I see this is working on Flutter Web app. It only doesn't work on Flutter macOS app. And [Raycast](https://www.raycast.com/) app seems to be available on macOS only.

yes, on windows and linux platforms there may be applications with similar functions

**Comment by justinmc** [comment]
Currently the [Clipboard](https://main-api.flutter.dev/flutter/services/Clipboard-class.html) class only supports plain text, I wonder if that is related? Otherwise not sure why this wouldn't work. I don't know how Raycast works but I'm assuming it's using the clipboard as usual.

**Comment by letungcntt** [comment]
> @letungcntt can you share a minimal reproduction that doesn't rely on downloading the app (since it will be tricky to debug in release mode)?

Sorry, I don't understand "minimal reproduction" you want, I have posted video above, can you specify again for me?

**Comment by huycozy** [comment]
@letungcntt I think "minimal reproduction" here means what event is triggered by Raycast app and sent to receiver (Flutter) app when pasting text. It would be helpful if you could find out and reproduce the issue without using an external app.

Maybe a Flutter code triggers clipboard service in an `unusual usage`? For eg, we usually get clipboard text with `Clipboard.getData(Clipboard.kTextPlain)` but assume that data format from Raycast is not kTextPlain (`text/plain`) as usual.

**Comment by knopp** [comment]
This is still an issue. It's reproducible. It's either something with our keyboard event handling, or absence of "Paste" action in Application menu.
